client/my-sites/sharing/connections/services/facebook.js

```diff
@@ -41,6 +45,7 @@ export class Facebook extends SharingService {
 										href="https://en.support.wordpress.com/publicize/#facebook-pages"
 										target="_blank"
 										rel="noopener noreferrer"
+										onClick={ this.recordLinkClickEvent }
```

Comment:
It's just a suggestion in order to be consistent with [other parts](https://github.com/Automattic/wp-calypso/blob/e80e3644c2185824280fbf75dd41cac41a1bc1ec/client/login/wp-login/login-links.jsx#L47) of our code, but we could name that method `recordNoticeLinkClick` instead.